wp-includes/sqlite/class-wp-sqlite-db.php

```diff
@@ -249,12 +287,12 @@ public function query( $query ) {
 		}
 
 		$this->last_error = $this->dbh->get_error_message();
-		if ( $this->last_error && ( ! defined( 'WP_INSTALLING' ) || ! WP_INSTALLING ) ) {
+		if ( $this->last_error ) { // && ( ! defined( 'WP_INSTALLING' ) || ! WP_INSTALLING ) ) {
```

Comment:
I suspect they used these checks to prevent the errors coming up during WP installation from halting the process. There are no errors now so we can remove them:

```suggestion
		if ( $this->last_error ) {
```